src/lang/Messages.properties

```diff
@@ -976,8 +976,9 @@ core.api.other.messagesHarById = Gets the HTTP messages with the given IDs, in H
 core.api.other.sendHarRequest = Sends the first HAR request entry, optionally following redirections. Returns, in HAR format, the request sent and response received and followed redirections, if any. The Mode is enforced when sending the request (and following redirections), custom manual requests are not allowed in 'Safe' mode nor in 'Protected' mode if out of scope.
 core.api.other.htmlreport = Generates a report in HTML format
 core.api.other.mdreport = Generates a report in Markdown format
-core.api.other.messageHar = Gets the message with the given ID in HAR format
+core.api.other.jsonreport = Generates a report in JSON format
 core.api.other.xmlreport = Generates a report in XML format
+core.api.other.messageHar = Gets the message with the given ID in HAR format
```

Comment:
Why did this move?

While consistency is far from perfect, the general rule is that keys be alphabetic.